[OSDOCS-20299]: CQA: HCP on bare metal disconnected (1 of 2) (#113856)
openshift/openshift-docs: modules/hcp-dc-tls-hosted.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-dc-tls-hosted_{context}"]
= Adding the registry CA to the compute nodes for the hosted cluster

[role="_abstract"]

> In order for the data plane compute nodes in the hosted cluster to be able to retrieve images from the private registry, you need to add the registry CA to the compute nodes.

Reviewer: ```suggestion
To ensure that the data plane compute nodes in the hosted cluster can to retrieve images from the private registry, you must add the registry certificate authority (CA) to the compute nodes.
```
